lab_1/main.py

```diff
@@ -4,56 +4,168 @@
 """
 
 
-def tokenize(text: str) -> list:
-    """
+#data = open('data.txt', encoding='utf-8').read()
+data = 'The weather is sunny, the man is happy. man man happy'
+def tokenize(text):
+    if type(text) != str:
+        return []
+    else:
+        lines = text.split('\n')
+        tokens = []
+        for words in lines:
+            words = words.split ()
+            for word in words:
+                word = word.lower()
+                word_new = ''
+                for letter in word:
+                    if letter.isalnum():# or letter in "'-/"
```

Comment:
a token cannot be a digit or contain digits (look in lab_work_1.md)
 "isalnum()" checks both letters and digits